### PR TITLE
Update collection-index.yml

### PR DESCRIPTION
**Pull request by ChristopherMacGown**
This PR adds my feature repository which provides maintained [mcfly](https://github.com/cantino/mcfly) and [direnv](https://github.com/direnv/direnv) as devcontainer features.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -238,3 +238,8 @@
   contact: https://github.com/csutter/devcontainer-templates/issues
   repository: https://github.com/csutter/devcontainer-templates
   ociReference: ghcr.io/csutter/devcontainer-templates
+- name: Convenient features
+  maintainer: ChristopherMacGown
+  contact: https://github.com/ChristopherMacGown/devcontainer-features/issues
+  repository: https://github.com/ChristopherMacGown/devcontainer-features
+  ociReference: ghcr.io/ChristopherMacGown/devcontainer-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/christophermacgown/devcontainer-features
```

image

The OCI spec accepts a lower case, hence the artifacts are published to same. Can we update the case here? 

ℹ️ The supported tooling would convert the ociReference to lowercase (if necessary), but probably better to fix it.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks, added one comment.

**Comment by ChristopherMacGown** [comment]
Thank you for the review, @samruddhikhandale I've made the changes you requested.

**Review by samruddhikhandale — APPROVED** [review]
⚡ ⚡

**Pull request merged**